astropy/table/jsviewer.py

```diff
@@ -32,9 +32,9 @@ class Conf(_config.ConfigNamespace):
 
 conf = Conf()
 
-
-EXTERN_JS_DIR = abspath(join(dirname(extern.__file__), "jquery", "data", "js"))
-EXTERN_CSS_DIR = abspath(join(dirname(extern.__file__), "jquery", "data", "css"))
+_TABLE_DIR = Path(extern.__file__).parent
+EXTERN_JS_DIR = _TABLE_DIR.joinpath("jquery", "data", "js").resolve()
+EXTERN_CSS_DIR = _TABLE_DIR.joinpath("jquery", "data", "css").resolve()
```

Comment:
I'm hesitant to even say it out loud, but are global constants like this considered part of the API? They are changing type from `str` to `Path`.

In practice it is probably OK and unlikely that any application code is using them.

Reply:
I'm taking a bet here, because the next version is major anyway. Would you rather I treat these as really public ?

Reply:
Being pragmatic, I think this is fine. Low risk and low impact.